Mathlib/Tactic/DeprecateTo.lean

```diff
@@ -43,9 +44,12 @@ open Lean Elab Term Command
 /-- Produce the syntax for the command `@[deprecated (since := "YYYY-MM-DD")] alias n := id`. -/
 def mkDeprecationStx (id : TSyntax `ident) (n : Name) (dat : Option String := none) :
     CommandElabM (TSyntax `command) := do
-  let dat := ← match dat with
-                | none => IO.Process.run { cmd := "date", args := #["-I"] }
-                | some s => return s
+  let dat := ←
+    match dat with
+      | none => do
+        let now ← Std.Time.ZonedDateTime.now
+        return Std.Time.Formats.leanDate.formatBuilder now.year now.month now.day
```

Comment:
```suggestion
        return s!"{(← Std.Time.ZonedDateTime.now).toPlainDate}"
```
I don't know `Std.Time.Format` very much, but you could also use this: at least, it works locally on Linux.